packages/blockchain/src/index.ts

```diff
@@ -900,16 +899,19 @@ export default class Blockchain implements BlockchainInterface {
   /**
    * @hidden
    */
-  _delBlock(blockHash: Buffer | typeof Block, cb: any) {
```

Comment:
Now that `Block` is a class, we can drop the `typeof` keyword.
https://github.com/ethereumjs/ethereumjs-vm/pull/682/files#diff-71a1f7e2560231b593e7d6e7805e230fL23